src/drivers/actuators/voxl_esc/qc_esc_packet.c

```diff
@@ -130,7 +132,7 @@ int32_t qc_esc_create_rpm_packet4_fb(int16_t rpm0, int16_t rpm1, int16_t rpm2, i
 				     uint8_t led0, uint8_t led1, uint8_t led2, uint8_t led3,
 				     int32_t fb_id, uint8_t *out, uint16_t out_size)
 {
-	uint16_t data[5];
+	int16_t data[5];
```

Comment:
I'm guessing that this isn't strictly needed but you wanted to fix it so that the data types match properly?